app/src/main/java/com/amaze/filemanager/ui/fragments/AppsListFragment.java

```diff
@@ -127,9 +127,6 @@ public boolean onOptionsItemSelected(@NonNull MenuItem item) {
       case R.id.sort:
         showSortDialog(((MainActivity) requireActivity()).getAppTheme());
         return true;
-      case R.id.exit:
```

Comment:
This fixes another issue AFAIK. Please add fixes.

Reply:
Updated PR description